settings/src/main/scala/org/typelevel/sbt/TypelevelSettingsPlugin.scala

```diff
@@ -209,15 +209,27 @@ object TypelevelSettingsPlugin extends AutoPlugin {
     scalacOptions ++= {
       val releaseOption = tlJdkRelease
         .value
-        .map { release => if (isJava8) Seq.empty else Seq("-release", release.toString) }
+        .map {
+          case 8 if isJava8 => Seq.empty
+          case n if n >= 8 => Seq("-release", n.toString)
+          case n => sys.error(s"'$n' is not a valid choice for '-release'")
+        }
```

Comment:
Sorry to nitpick :(

This doesn't seem very DRY :) also I wonder if an error message specifically referencing the `tlJdkRelease` setting might do more to point users in the right direction. Also, there's a difference between "your JDK is too old" vs "n < 8 is not a valid choice".

Thanks again for your work on this ❤️

Reply:
Done.